webapp/src/Controller/API/GroupController.php

```diff
@@ -66,6 +70,44 @@ public function singleAction(Request $request, string $id)
         return parent::performSingleAction($request, $id);
     }
 
+    /**
+     * Add a new group
+     *
+     * @param Request             $request
+     * @param ImportExportService $importExport
+     *
+     * @return Response
+     *
+     * @Rest\Post()
+     * @Security("is_granted('ROLE_ADMIN') or is_granted('ROLE_API_WRITER')")
+     * @OA\RequestBody(
+     *     required=true,
+     *     @OA\MediaType(
+     *         mediaType="multipart/form-data",
+     *         @OA\Schema(ref="#/components/schemas/TeamCategory")
+     *     ),
+     *     @OA\MediaType(
+     *         mediaType="application/json",
+     *         @OA\Schema(ref="#/components/schemas/TeamCategory")
+     *     )
+     * )
+     * @OA\Response(
+     *     response="200",
```

Comment:
Why should this not be 201? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

Reply:
201 might make sense. We should then also set the Location header